etl/steps/data/garden/un/2025-03-12/migrant_stock.py

```diff
@@ -0,0 +1,257 @@
+import pandas as pd
+from owid.catalog import Table
+from owid.catalog import processing as pr
+
+from etl.data_helpers import geo
+from etl.helpers import PathFinder
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+# Column names for the years are different by sex
+BOTH_SEXES = ["_1990", "_1995", "_2000", "_2005", "_2010", "_2015", "_2020", "_2024"]
+MALES = ["_1990_1", "_1995_1", "_2000_1", "_2005_1", "_2010_1", "_2015_1", "_2020_1", "_2024_1"]
+FEMALES = ["_1990_2", "_1995_2", "_2000_2", "_2005_2", "_2010_2", "_2015_2", "_2020_2", "_2024_2"]
+
+ALL_YEARS = BOTH_SEXES + MALES + FEMALES
+
+
+def run() -> None:
+    #
+    #
+    # Load inputs.
+    #
+    # Load meadow dataset.
+    ds_meadow = paths.load_dataset("migrant_stock")
+    # Read tables from meadow dataset.
+    # destination and origin table
+    tb_do = ds_meadow.read("migrant_stock_dest_origin")
+    # destination table, total numbers and shares
+    tb_d_total = ds_meadow.read("migrant_stock_dest_total")
+    tb_d_share = ds_meadow.read("migrant_stock_dest_share")
+    tb_d_pop = ds_meadow.read("un_desa_total_population")
+    # origin table
+    tb_o = ds_meadow.read("migrant_stock_origin")
+
+    # Australia & New Zealand shows up twice in the data
+    duplicates = tb_do[tb_do.duplicated(subset=["country_destination", "country_origin"])]
+    assert (
```

Comment:
I would just omit Australia and New Zealand from the datasets, we tend to try and avoid having too many non-OWID entities in the dataset as it can get a bit out of hand.